src/js/comments.test.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { commentCounter } from './commentCounter';
+
```

Comment:
- [ ] It is good practice to test your counter for possible errors. (What happens if the array is empty, What happens if I pass a string instead of an array, etc...)